cloud/README.md

```diff
@@ -1,3 +1,126 @@
 # Gen3 Cloud Native EKS Setup
 
-TODO
+Follow general procedure outlined here:
+https://github.com/uc-cdis/cloud-automation/blob/master/doc/csoc-free-commons-steps.md
+
+
+These are notes on the UMCCR specific setup. It is follows closely the instructions set out in the above mentioned documentation, with minor adjustments where they were needed.
+
+
+## Part 1 Admin VM
+
+AWS EC2 admin server
+
+UMCCR setup:
+
+- Ubuntu 18.04 server (issues with newer versions or other OS)
+- t2.meduim
+- min 10GB disk (should probably be a bit more)
+- hibernation enabled (could switch to creating new instances and recover data/config)
+- admin like IAM role (`gen3-admin-vm-instance-role`)
+- no SSH key -> use SSM
+- SG (`gen3-sg`) to allow inbound SSH
+
+
+Instance setup
+```bash
+# work under the system's ubuntu user
+sudo su ubuntu
+# make sure the user has full access/control of required folders
+sudo chown ubuntu:ubuntu ~/.aws ~/.local
+```
+
+
+Follow the Gen3 setup steps
+```bash
+# working outside the users $HOME caused issues (as it is assumed in various scripts)
+cd $HOME
+git clone https://github.com/uc-cdis/cloud-automation.git
+export GEN3_HOME="$HOME/cloud-automation"
+export GEN3_NOPROXY='no'
+# be careful to retain the env vars when running the setup script
+sudo -E bash cloud-automation/gen3/bin/kube-setup-workvm.sh
+
+# Double check the entries created in the .bashrc
+vim ${HOME}/.bashrc
+source ${HOME}/.bashrc
+
+# prepare the AWS profiles
+echo "[default]
+output = json
+region = ap-southeast-2
+credential_source = Ec2InstanceMetadata
```

Comment:
Remember, we have to comment out this line to work with terraform 0.12 aws provider (the respective terraform module config didn't lock the aws provider version, hence, it pulls latest). 
```
#credential_source = Ec2InstanceMetadata
```

Otherwise, we get this error:
```
Error: error configuring Terraform AWS Provider: Error creating AWS session: CredentialRequiresARNError
```

https://www.google.com/search?q=Error%3A+error+configuring+Terraform+AWS+Provider%3A+Error+creating+AWS+session%3A+CredentialRequiresARNError

Reply:
Ah, yes. Good point!